mlflow-tests/tests/test_models.py

```
@@ -0,0 +1,159 @@
+        mlflow.set_workspace(self.test_context.active_workspace)
+        logger.info(f"Set active workspace to: {test_data.workspace_to_use}")
+        logger.debug(f"Created authenticated MLflow client for user: {user_info.uname}")
+
+        # Step 4: Execute action if provided
+        if test_data.action_func:
+            action_name = test_data.action_func.__name__
+            logger.info(f"Step 4: Executing action '{action_name}'")
+            try:
+                test_data.action_func(self.test_context)
+                logger.info(f"Action '{action_name}' completed without exception")
+            except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)